met-api/tests/utilities/factory_scenarios.py

```diff
@@ -36,7 +36,10 @@ class TestEngagemntInfo(dict, Enum):
         'description': 'My Test Engagement Description',
         'start_date': datetime.datetime.now().strftime('%m/%d/%Y, %H:%M:%S'),
         'end_date': datetime.datetime.now().strftime('%m/%d/%Y, %H:%M:%S'),
-        'rich_description': 'Rich Description Sample'
+        'rich_description': 'Rich Description Sample',
+        'content': 'Content Sample',
+        'rich_content': 'Rich Content Sample',
```

Comment:
I thought this was a Json type?!

Reply:
'rich_description': 'Rich Description Sample', has been there for a while in main, this is just for the unit tests and it would be better to add a json string here totally agreed. We can create a bug/enhancement task for this separately as it isn't part of this PR. And the 'rich_content': 'Rich Content Sample', has been pushed there with the api PR

Reply:
Bug issue task for this created